src/components/Input/Input.tsx

```diff
@@ -6,13 +6,19 @@ import "../../styles/tokens.css";
 import { sanitizeForId } from "../../utils";
 import { LabelPosition } from "../../types";
 import { Icon } from "../icon";
+import React from "react";
 
+/** Props for the Input */
 export type InputProps = {
   id: string;
   name: string;
+  type: 'text' | 'number';
```

Comment:
Notice how we extend the prop type defintion of the component, and spread {...props}. The idea was that we don't have to expose functions for already native functionality.

Reply:
Right. Moving the numerical functionality to specific type Numeric and leaving the input in peace.